pyproject.toml

```diff
@@ -118,6 +118,10 @@ test-extras = ["test"]
 test-command = "pytest {project}/tests -v"
 test-skip = ["*-musllinux*", "cp312-win*", "*-macosx_arm64"]
 
+[tool.cibuildwheel.macos]
+# https://cibuildwheel.readthedocs.io/en/stable/faq/#apple-silicon
+archs = ["x86_64", "arm64"]
```

Comment:
As you use macos-13 and macos-latest this change is obsolete

Reply:
thanks :) yep, still cleaning up here